acs-integration-tests/pom.xml

```diff
@@ -621,6 +629,56 @@
         </profile>
 
         <profile>
+            <id>redis</id>
+            <dependencies>
```

Comment:
Added changes to `AcsTestsReadme.md` in the master branch of the acs-ci-config Git repo. Also these changes to add the `redis` Maven profile have been pulled out to another branch but when I re-introduce it I'll look into how to standardize the test output directories.